docs/UserGuide.md

```diff
@@ -72,11 +72,43 @@ Tinner aims to solve the hassle of managing internship applications though 3 asp
 --------------------------------------------------------------------------------------------------------------------
 # About <a id="about"></a>
 
-This section helps you to understand the Graphical User Interface (GUI), commonly used technical terminologies, general symbols and information about the command format. 
-
 ## Outline of this Guide <a id="outline"></a>
+The User Guide's primary goal is to assist the user in learning how to use the application Tinner
```

Comment:
I feel Iike this could be segmented. Describing the “about” segment could be done above. Then describing the general flow of the guide could be done here

Reply:
Agreed, moved this segment under About